### DISCUSSION TITLE
Enabling GPS tab on CFClient

### DISCUSSION THREAD

**Discussion by SkywalkerJLi** [bitcraze/discussions#812] [Q&A] [answered]
Hi!

I am currently running cfclient on my Mac M1 computer with python 3.11 and am trying to attach a GPS sensor to the CrazyFlie like this forum post: https://www.bitcraze.io/2014/03/crazyflie-with-gps-round-2/. I noticed, however, that the GPS tab is not showing up on the client even though the code, `GpsTab.py and gpsTab.ui`, is there within `/usr/local/lib/python3.11/site-packages/cfclient/ui/tabs`. Has anyone else gotten a gps module to work with the CrazyFlie? Would love to hear how they did it!

Thanks!

**Comment by ataffanel** [comment, marked as answer]
This tab has not been used not maintained for a very long time. So, it has been disabled by default by commenting it our in __init__.py: https://github.com/bitcraze/crazyflie-clients-python/blob/master/src/cfclient/ui/tabs/__init__.py#L36C14-L36C14. 

If you uncomment the line 36 and 55 of `tabs/__init__.py` you will re-enable the tab. There will likely be some work involved to get it to work again though.

**Comment by SkywalkerJLi** [comment]
Thanks! That did the trick.
